WooCommerce/Classes/ViewRelated/Blaze/BlazeCampaignList/BlazeCampaignListViewModel.swift

```diff
@@ -100,8 +104,14 @@ final class BlazeCampaignListViewModel: ObservableObject {
         }
     }
 
-    func didSelectCampaignDetails() {
+    func didSelectCampaignDetails(_ campaign: BlazeCampaign) {
         analytics.track(event: .Blaze.blazeCampaignDetailSelected(source: .campaignList))
+
+        let path = String(format: Constants.campaignDetailsURLFormat,
+                          campaign.campaignID,
+                          siteURL.trimHTTPScheme(),
```

Comment:
❓ will this still work if `siteUrl` is `""` ?

Reply:
Yes, if you look at the implementation of the method, you see it returns a URL without scheme if possible, or the initial string.